v2/routing_test.go

```diff
@@ -0,0 +1,36 @@
+package dht
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/libp2p/go-libp2p/core/crypto"
+	"github.com/libp2p/go-libp2p/core/peer"
+	"github.com/stretchr/testify/require"
+
+	"github.com/libp2p/go-libp2p-kad-dht/v2/internal/kadtest"
+)
+
+func TestGetSetValueLocal(t *testing.T) {
+	ctx := kadtest.CtxShort(t)
+
+	top := NewTopology(t)
+	d := top.AddServer(nil)
+
+	_, pub, _ := crypto.GenerateEd25519Key(rng)
+	v, err := crypto.MarshalPublicKey(pub)
+	require.NoError(t, err)
+
+	id, err := peer.IDFromPublicKey(pub)
+	require.NoError(t, err)
```

Comment:
in `handlers_test.go` is a `newPeerID` function

Reply:
nice. I'd like to consolidate all these helpers